splinterd/src/daemon.rs

```diff
@@ -510,8 +526,8 @@ impl SplinterDaemon {
             Box::new(AllowAllKeyPermissionManager),
             Some(self.admin_timeout),
             routing_writer.clone(),
-            #[cfg(feature = "admin-service-event-store")]
-            admin_event_store,
+            #[cfg(feature = "service-arg-validation")]
```

Comment:
This should be admin-service-event-store

Reply:
Fixed